### PR TITLE
Test: run tox (pytest, flake8) on CI

### LINKED ISSUE #9: Set up ci

### PR DESCRIPTION
**Pull request by carrascomj**
Closes #9 

### Description

This PR adds a CI with [tox](https://pypi.org/project/tox/) ([flake8](https://pypi.org/project/flake8), [pytest](https://pypi.org/project/pytest/)) which runs everytime a commit is pushed to master, on PRs to master and every Monday.

### Implementation

The monday scheduling can be _safely_ removed, only a non-pinned dependency of an explicit dependency or click should be able to silently break CI.

* A github action was created that runs tox with pytest and flake8 (taken from Maud).
* The test data was updated with the new format used in maud 0.3.2.
* The `setup.cfg` was changed to make flake8 [black](https://pypi.org/project/black/)-compliant and a "development" extra_requires was added. `isort` and `black` were run too.

I have **not** added an action for releases yet, but it might be better to leave that for another PR (?).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+rc[0-9]+'
```
Comment (teddygroves) [inline .github/workflows/ci.yml:8]: possibly whatever kind of regex this is already does this but I guess we might need to escape the `.`s here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install Maud
```
Comment (teddygroves) [inline .github/workflows/ci.yml:42]: I think this should be called "Install maudtools"

**Review by teddygroves — CHANGES_REQUESTED** [review]
Looks good, just a small question about the regex for tags and a name change.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (carrascomj) [reply .github/workflows/ci.yml]: Ups, yes.

**Review by teddygroves — APPROVED** [review]

**Pull request merged**